doc/devdocs/modules/powerpreview/monaco/readme.md

```diff
@@ -0,0 +1,15 @@
+# Developer Preview (Monaco)
+## Update monaco editor
+
+1. Download Monaco editor with npm: `npm i monaco-editor`.
+2. Delete everything except the `min` folder (the minimised code).
+3. Copy the `min` folder inside the [`monacoSRC`](/src/modules/previewpane/MonacoPreviewHandler/monacoSRC) folder.
+4. Generate the JSON file (see section below)
+
+## Generate JSON file
```

Comment:
What does this json?

Reply:
The installer reads from this which file extensions it should register and the FileHandler class reads from it the monaco id of the language

Reply:
I mean we should describe this in the docs.